packages/okta-react/README.md

```diff
@@ -179,30 +179,21 @@ Security is the top-most component of okta-react. This is where most of the conf
 
 #### Configuration options
 
+These options are used by `Security` to configure the [Auth](https://github.com/okta/okta-oidc-js/blob/master/packages/okta-react/src/Auth.js) object. The most commonly used options are shown here. See [Configuration Reference](https://github.com/okta/okta-auth-js#configuration-reference) for an extended set of supported options.
+
 - **issuer** (required) - The OpenId Connect `issuer`
-- **client_id** (required) - The OpenId Connect `client_id`
-- **redirect_uri** (required) - Where the callback handler is hosted
+- **clientId** (required) - The OpenId Connect `client_id`
```

Comment:
I'm 90% sure I know the answer, but what happens to users of the old standard?

If we're deprecating the old snake_case method, we should indicate that somewhere so users understand why their code works (per above) when it doesn't match the docs.  The changelog would satisfy me on that if it shows up there (and thus not in this PR), but we need to make sure it does show up there in that case.

Reply:
I will add note to the changelog